packages/module/src/ToolCall/ToolCall.scss

```diff
@@ -1,6 +1,6 @@
 .pf-chatbot__tool-call {
-  --pf-v6-c-card--BorderColor: var(--pf-t--global--border--color--control--read-only);
   --pf-v6-c-card--BorderRadius: var(--pf-t--global--border--radius--small);
+  --pf-v6-c-card--BorderColor: var(--pf-t--global--border--color--control--read-only);
```

Comment:
i wonder if these should just use our border--color--default token instead of read-only? they would get the HC shade automatically if so. 

@kaylachumley is there a reason why this is using the border--color--control--read-only token?

Reply:
Updated color! Was able to drop some styles.